examples/build/board_build_example.py

```diff
@@ -10,19 +10,7 @@
 import rhea.build as build 
 from rhea.build.boards import get_board
 from rhea.build.boards import get_all_board_names
-#from blink import blinky
-
-# some boards don't have LEDs but they have IO that
-# can drive LEDs.  The following maps the led port 
-# to a set of pins for boards without an led port.
-led_port_pin_map = {
-    'xula':  dict(name='led', pins=(36, 37, 50, 52)),
-    'xula2': dict(name='led', pins=('R7', 'R15', 'R16', 'M15',)),
-    'pone': dict(name='led', pins=(18, 23, 26, 33)),
-    # the de0-cv led port is named "ledr", for the generic testing
-    # add the led port
-    'de0cv': dict(name='led', pins=('AA2', 'AA1', 'W2', 'Y3',))
-}
+from rhea.build.boards import led_port_pin_map
```

Comment:
The comment was on the import, you didn't add anything to `rhea.build.boards` the import fails, the import needs to change to `from test_boards import led_port_pin_map`.  Not sure if that was the goal of your changes?